app/controllers/api/v1/subscriptions_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Api::V1::SubscriptionsController < ApplicationController
+  before_action :authenticate_user!
+  rescue_from Stripe::InvalidRequestError, with: :invalid_token
```

Comment:
there must be other potential errors that can happen here?

Reply:
added another potential error